setup.py

```diff
@@ -25,65 +25,66 @@ def load_version():
-              'Intended Audience :: Developers',
-              'License :: OSI Approved',
-              'Programming Language :: C',
-              'Programming Language :: Python',
-              'Topic :: Software Development',
-              'Topic :: Scientific/Engineering',
-              'Operating System :: Microsoft :: Windows',
-              'Operating System :: POSIX',
-              'Operating System :: Unix',
-              'Operating System :: MacOS',
-              'Programming Language :: Python :: 3.7',
-              'Programming Language :: Python :: 3.8',
-              'Programming Language :: Python :: 3.9',
-              'Programming Language :: Python :: 3.10',
-          ],
-          packages=find_packages(),
-          package_data={
-              'nilearn.datasets.data': ['*.nii.gz', '*.csv', '*.txt'],
-              'nilearn.datasets.data.fsaverage5': ['*.gz'],
-              'nilearn.surface.data': ['*.csv'],
-              'nilearn.plotting.data.js': ['*.js'],
-              'nilearn.plotting.data.html': ['*.html'],
-              'nilearn.plotting.glass_brain_files': ['*.json'],
-              'nilearn.tests.data': ['*'],
-              'nilearn.image.tests.data': ['*.mgz'],
-              'nilearn.surface.tests.data': ['*.annot', '*.label'],
-              'nilearn.datasets.tests.data': ['*.*'],
-              'nilearn.datasets.tests.data.archive_contents': ['*'],
-              'nilearn.datasets.tests.data.archive_contents.nyu_rest': ['*'],
-              'nilearn.datasets.tests.data.archive_contents.test_examples':
-              ['*'],
-              'nilearn.datasets.description': ['*.rst'],
-              'nilearn.reporting.data.html': ['*.html'],
-              'nilearn.glm.tests': ['*.nii.gz', '*.npz'],
-              'nilearn.reporting.glm_reporter_templates': ['*.html'],
-          },
-          )
+    setup(
+        name=DISTNAME,
+        maintainer=MAINTAINER,
+        maintainer_email=MAINTAINER_EMAIL,
+        description=DESCRIPTION,
+        license=LICENSE,
+        url=URL,
+        version=VERSION,
+        download_url=DOWNLOAD_URL,
+        long_description=LONG_DESCRIPTION,
+        zip_safe=False,  # the package can run out of an .egg file
+        classifiers=[
+            "Intended Audience :: Science/Research",
+            "Intended Audience :: Developers",
+            "License :: OSI Approved",
+            "Programming Language :: C",
+            "Programming Language :: Python",
+            "Topic :: Software Development",
+            "Topic :: Scientific/Engineering",
+            "Operating System :: Microsoft :: Windows",
+            "Operating System :: POSIX",
+            "Operating System :: Unix",
+            "Operating System :: MacOS",
+            "Programming Language :: Python :: 3.7",
+            "Programming Language :: Python :: 3.8",
+            "Programming Language :: Python :: 3.9",
+            "Programming Language :: Python :: 3.10",
```

Comment:
We can add Python `3.11`

Reply:
Actually could this be the occasion to move most of that metadata in the setup.cfg?

Reply:
@Remi-Gau maybe leave for now? This might change soon anyways as we discussed changing our packaging implementation at the last coredev meeting. We can follow up on this topic at the next meeting.

Reply:
okidoki

Reply:
python 3.11 added